const.go

```diff
@@ -2,7 +2,7 @@ package cptv
 
 const (
 	magic        = "CPTV"
-	version byte = 0x01
+	version byte = 0x02
```

Comment:
There's probably no need to bump the version number. The CPTV file format is designed to that readers can easily skip over fields they don't no. The version byte is more about major structural changes to the file format.